yarn-project/aztec.js/src/fee/fee_juice_payment_method_with_claim.ts

```diff
@@ -3,34 +3,25 @@ import { Fr } from '@aztec/foundation/fields';
 import { ProtocolContractAddress } from '@aztec/protocol-contracts';
 import { FunctionSelector, FunctionType } from '@aztec/stdlib/abi';
 import type { AztecAddress } from '@aztec/stdlib/aztec-address';
+import type { GasSettings } from '@aztec/stdlib/gas';
 
 import type { L2AmountClaim } from '../ethereum/portal_manager.js';
-import { FeeJuicePaymentMethod } from './fee_juice_payment_method.js';
-
-/* eslint-disable @typescript-eslint/no-unsafe-declaration-merging */
-
-/** AppConfigurableFeePaymentMethod branding */
-export interface FeeJuicePaymentMethodWithClaim {
-  /** Brand. */
-  _branding: 'AppConfigurableFeePaymentMethod';
-}
+import type { FeePaymentMethod } from './fee_payment_method.js';
 
 /**
  * Pay fee directly with Fee Juice claimed on the same tx.
  */
```

Comment:
```suggestion
/**
 * Pay fee directly with Fee Juice claimed in the same tx. Claiming consumes an L1 to L2 message that "contains"
 * the fee juice bridged from L1.
 */
```
Unrelated to your PR but found the docs here a bit too concise.